src/Server/Services/Http/InferenceController.cs

```diff
@@ -0,0 +1,97 @@
+﻿/*
+ * Apache License, Version 2.0
+ * Copyright 2019-2020 NVIDIA Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Nvidia.Clara.DicomAdapter.API;
+using Nvidia.Clara.DicomAdapter.API.Rest;
+using Nvidia.Clara.DicomAdapter.Common;
+using Nvidia.Clara.DicomAdapter.Server.Common;
+using Nvidia.Clara.DicomAdapter.Server.Repositories;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Nvidia.Clara.DicomAdapter.Server.Services.Http
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class InferenceController : ControllerBase
+    {
+        private readonly IKubernetesWrapper _kubernetesClient;
+        private readonly ILogger<InferenceController> _logger;
+        private readonly IJobs _jobsApi;
+
+        public InferenceController(
+            IKubernetesWrapper kubernetesClient,
+            ILogger<InferenceController> logger,
+            IJobs jobsApi)
+        {
+            _kubernetesClient = kubernetesClient ?? throw new System.ArgumentNullException(nameof(kubernetesClient));
```

Comment:
we've got `using System;` at the top of the document, I don't think it's necessary to use the full name to reference `System.ArgumentNullException` here.

Reply:
Will be fixed in upcoming PR.